support-frontend/assets/helpers/urls/routes.ts

```diff
@@ -88,16 +88,28 @@ function paperSubsUrl(
 	return baseURL;
 }
 
-function digitalSubscriptionLanding(
+function digitalPlusCheckoutDeepLink(
```

Comment:
Would it be worth making clear in the name that this is _from_ the subscribe landing page (and not from the 3 tier landing page)?